.github/workflows/pr.yaml

```diff
@@ -1,124 +1,157 @@
         echo "Current directory:`pwd`"
-        echo "Folders:`ls`"
-        kubeplus_folder="$(basename `pwd`)"
-        echo "KubePlus folder name:$kubeplus_folder"
         mkdir -p $HOME/go/src/github.com/cloud-ark
         cd ..
-        runner_dir=`pwd`
         echo "Current directory:`pwd`"
-        mv $kubeplus_folder $HOME/go/src/github.com/cloud-ark/kubeplus
+        kubeplus_folder="$(basename `pwd`)"
+        echo "KubePlus folder name:$kubeplus_folder"
+        cp -R $kubeplus_folder $HOME/go/src/github.com/cloud-ark/kubeplus
         cd $HOME/go/src/github.com/cloud-ark/kubeplus
+        export KUBEPLUS_NS=default
+        echo "KUBEPLUS_NS=default" >> $GITHUB_ENV
         export KUBEPLUS_HOME=`pwd`
+        echo "KUBEPLUS_HOME=$KUBEPLUS_HOME" >> $GITHUB_ENV
         export PATH=$KUBEPLUS_HOME/plugins:$PATH
-        echo "PATH:$PATH"
-        echo "KUBEPLUS_HOME:$KUBEPLUS_HOME"
-        kubectl kubeplus commands
-        export KUBEPLUS_NS=default
-        python3 -m venv venv
-        source venv/bin/activate
-        pip3 install -r requirements.txt
-        apiserver=`kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}'`
-        echo "API_SERVER_URL:$apiserver"
-        python3 provider-kubeconfig.py -s $apiserver create $KUBEPLUS_NS
-        deactivate
+        echo "PATH=$PATH" >> $GITHUB_ENV
 
-        echo "Building mutating-webhook..."
+    - name: Build Mutating Webhook
+      run: |
+        echo "KUBEPLUS_HOME:$KUBEPLUS_HOME"
         cd $KUBEPLUS_HOME/mutating-webhook
-        export GO111MODULE=on; go get github.com/googleapis/gnostic@v0.4.0
+        export GO111MODULE=on
+        go get github.com/googleapis/gnostic@v0.4.0
         ./build-artifact.sh latest
 
-        echo "Building helmer..."
+    - name: Build Helmer
+      run: |
         cd $KUBEPLUS_HOME/platform-operator/helm-pod/
         go mod vendor
         ./build-artifact.sh latest
 
-        echo "Building platform-operator..."
+    - name: Build Platform Operator
+      run: |
         cd $KUBEPLUS_HOME/platform-operator
         ./build-artifact.sh latest
 
-        echo "Building kubeconfiggenerator..."
+    - name: Build KubeConfig Generator
+      run: |
         cd $KUBEPLUS_HOME/deploy
         ./build-artifact-kubeconfiggenerator.sh latest
-        #echo "Building webhook_init_container..."
-        #./build-artifact.sh latest
-        #echo "Building resource cleaner..."
-        #./build-artifact-clean.sh latest
-
-        #cd $KUBEPLUS_HOME/consumerui
-        #echo "Building consumer ui..."
-        #./build-artifact.sh latest
 
-        cd $KUBEPLUS_HOME
-        ls
-        docker images
+    - name: Load Docker Images to Kind cluster
+      run: |
+        for image in $(docker images --format '{{.Repository}}:{{.Tag}}' | grep '^gcr.io/cloudark-kubeplus'); do
+          kind load docker-image "$image" --name $CLUSTER_NAME
+        done
+        docker exec $CLUSTER_NAME-control-plane crictl images
```

Comment:
This ensures that all Docker images tagged with `gcr.io/cloudark-kubeplus` (helmer-pod, kubeconfig-generator, platform)  are loaded into the Kind cluster so that we can use them in kubeplus installation.